src/login/logind-button.c

```diff
@@ -77,10 +112,27 @@ int button_set_seat(Button *b, const char *sn) {
         return free_and_strdup(&b->seat, sn);
 }
 
-static void button_lid_switch_handle_action(Manager *manager, bool is_edge) {
+static ButtonModifierMask button_get_keycode_modifier_mask(int keycode)
+{
+        ButtonModifierMask modifier_mask = BUTTON_MODIFIER_NONE;
+
+        assert(keycode);
+
+        FOREACH_ARRAY(keycode_modifier, keycode_modifier_table, ELEMENTSOF(keycode_modifier_table)) {
```

Comment:
we usually give local variables short names. hence maybe rename the first arg here to `i` or so (for "iterator")